homeassistant/components/apple_tv/config_flow.py

```diff
@@ -34,6 +34,7 @@
     SchemaFlowFormStep,
     SchemaOptionsFlowHandler,
 )
+from homeassistant.helpers.service_info.zeroconf import ZeroconfServiceInfo
```

Comment:
Note: zeroconf is still being imported.
It is marked as a dependency in manifest.py